vividus/src/main/java/org/vividus/bdd/expression/EvalExpressionProcessor.java

```diff
@@ -36,7 +37,8 @@
     private static final Pattern EVAL_PATTERN = Pattern.compile("^eval\\((.*)\\)$", Pattern.CASE_INSENSITIVE
             | Pattern.DOTALL);
     private static final int EVAL_GROUP = 1;
-    private static final Map<String, Object> NAMESPACES = Map.of("math", Math.class, "stringUtils", StringUtils.class);
+    private static final Map<String, Object> NAMESPACES = Map.of("math", Math.class, "stringUtils", StringUtils.class,
+            "wordUtils", WordUtils.class);
```

Comment:
I doubtful we need to add more and more utils here, what about just new simple expression?

Reply:
Why not?
I see only one problem regarding the syntax of such logic, but for me, flexibility of the code worth it